production/loki-mixin/config.libsonnet

```diff
@@ -17,6 +17,9 @@
     per_namespace_label: 'namespace',
     per_job_label: 'job',
 
+    // The Log Formater that is used within the Dashboards (logfmt,json)
+    log_format: 'json',
```

Comment:
I think the default would need to be 'logfmt' to maintain backwards compatibility

Reply:
oh sorry. That was something i didn't want to push. 

:D Used the same repo for creating our Dashboards and we use json :D